test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5566,6 +5590,44 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
                 rescs = []
             if len(rescs) > 0:
                 self.manager(MGR_CMD_DELETE, RSC, id=rescs, expect=True)
+            if self.platform == 'cray' or self.platform == 'craysim':
+                nodes = []
```

Comment:
I think you don't need this code as well also. To revert vnodes just add following code in pbspro/test/fw/ptl/utils/pbs_testsuite.py at line no 925.
```
if mom.platform == 'cray' or mom.platform == 'craysim':
    try:
        self.server.manager(MGR_CMD_DELETE, NODE, None, '')
    except:
        pass
```